src/Options.ts

```diff
@@ -18,9 +18,15 @@ const options = {
   headless: {
     alias: 'h',
     default: true,
-    description: 'Wether the browser should be ran "headfully" (non-headlessly)',
+    description: 'Whether the browser should be ran "headfully" (non-headlessly)',
     type: 'boolean' as const,
   },
+  pattern: {
+    alias: 'p',
+    default: /[\s\S]*/,
```

Comment:
Would it be equivalent to directly target any number of any character - `/.*/` ?

Reply:
yeah definitely I prematurely psyched myself out https://stackoverflow.com/questions/6711971/regular-expressions-match-anything

Reply:
Ah, I actually had no idea that `.` doesn't match newlines. That's interesting.